ibis/sql/compiler.py

```diff
@@ -1254,8 +1254,16 @@ def _category_label(expr):
     return stmt.end().name(expr._name)
 
 
+@rewrites(ops.Sum)
```

Comment:
@wesm I was trying to do a test to use rewrites decorator for SUM. why does it not get called?